src/components/views/auth/signIn/SignIn.spec.js

```diff
@@ -16,54 +16,69 @@ const props = {
 describe('SignIn Component', () => {
   it('should render without errors', () => {
     const component = shallow(
-      <Router>
-        <Signin store={store} {...props} />
-      </Router>,
+      <Provider store={store}>
```

Comment:
I must have missed this when I reviewed the signup/login PR, I think we should use a mock store for testing.

Also, why did you now mount a `Provider`?